lib/tasks/docker.rake

```diff
@@ -78,6 +78,10 @@ def migrate_databases(parallel: false, load_plugins: false)
   success
 end
 
+def system_tests_parallel_tests_processors_env
+  "PARALLEL_TEST_PROCESSORS=#{Etc.nprocessors / 2}"
```

Comment:
Note that we are only using half the CPU cores to leave room for the chrome processes that will be spawn during system tests.

Reply:
Is half enough or do we need to do (n/2-1)? I feel like I recall something like this before..